rio_tiler/utils.py

```diff
@@ -206,6 +207,15 @@ def get_vrt_transform(
     vrt_height = max(1, round((s - n) / h_res))
     vrt_transform = from_bounds(w, s, e, n, vrt_width, vrt_height)
 
+    # Warns when the read resolution (tile, part) is lower than the
+    # resolution of the smaller overview. This will result in rasterio/rio-tiler
+    # creating a big WarpedVRT and trying to read a small part of it.
```

Comment:
I'd put this in the second line of the warning message as well. Just saying

> Trying to fetch lower resolution than min overview resolution

doesn't really tell anything to the user, and as a user I'd be confused why I'm getting warned at.

Reply:
yeah, TBH I'm still not sure about the warning, I hope I'll be able to find a solution instead 😄